lib/workload/components/api-gateway/README.md

```diff
@@ -0,0 +1,26 @@
+# ApiGatewayConstruct
+
+Usage example:
+
+```ts
+const srmApi = new ApiGatewayConstruct(this, 'ApiGateway', props.apiGatewayCognitoProps);
+const httpApi = srmApi.httpApi;
```

Comment:
Should `srmApi` not be `apiGateway` to match the code below?

Reply:
Ahh yes, was copy-paste here and there. Thanks!

Reply:
Thought so. All good!